ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -441,12 +440,16 @@ def record_process_sample(project, sample, workflow_subtask, analysis_module_nam
         if workflow_subtask == "merge_process_variantcall":
             sample_status_field = "analysis_status"
             sample_status_value = "UNDER_ANALYSIS"
+            sample_data_status_field = "status"
+            sample_data_status_value = "STALE"
             seqrun_status_field = "alignment_status"
             seqrun_status_value = "RUNNING"
             extra_args = {"mean_autosomal_coverage": 0}
         elif workflow_subtask == "genotype_concordance":
             sample_status_field = seqrun_status_field = "genotype_status"
             sample_status_value = seqrun_status_value = "UNDER_ANALYSIS"
+            sample_data_status_field = "status"
+            sample_data_status_value = "STALE"
```

Comment:
To be honest, I have no idea how the genotype analysis is run. I decided to have the same behaviour as the other analysis for coherence, but I'm really not sure wether it makes any sense.